modules/ztp-sno-du-disk-encryption.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-reference-cluster-configuration-for-vdu.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-sno-du-disk-encryption_{context}"]
+= About disk encryption with PCR protection
+
+You can configure disk encryption by using the `diskEncryption` field in the `SiteConfig` custom resource (CR).
+
+When you enable disk encryption, the disks are encrypted with Platform Configuration Registers (PCRs) protection. PCR within the TPM store hash values that represent the current hardware and software configuration of your system. You can use the following PCR registers:
+
+* PCR 1: Represents the Unified Extensible Firmware Interface (UEFI) state.
+* PCR 7: Represents the secure boot state.
+
+
+The Trusted Platform Module (TPM) is a component that stores cryptographic keys and evaluates the security state of your system. 
+ TPM safeguards encryption keys by linking them to the system's current state, as recorded in PCR 1 and PCR 7. The `dmcrypt` utility uses these keys to encrypt the disk. TPM uses these PCR values to unlock the encryption keys, which are then used to decrypt the disk whenever required. If the server is moved or tampered with, the TPM PCR keys cannot be replicated, and the disk remains encrypted and inaccessible.
+
+For more information about TPM, see "TPM encryption".
+
+[NOTE]
+====
```

Comment:
Use the TP snippet for this, for example:
```
:FeatureName: Configuring disk encryption by using the `diskEncryption` field in the `SiteConfig` CR
include::snippets/technology-preview.adoc[]
:!FeatureName:
```

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#indicating-technology-preview-features

Reply:
Updated as per the suggestion.